### DISCUSSION TITLE
Should we use `--pool ` with celery workers by default in helm chart?

### DISCUSSION THREAD

**Discussion by shakeelansari63** [apache/superset#27070] [Ideas]
I was deploying Superset on Azure AKS and noticed that the celery worker pods were continuously crashing with OOM killed message.     
   
For long time I could not identify why AKS keeps killing my Worker pods. They weren't using lot of memory.   
Then I came across this [Gist](https://gist.github.com/chris-w-jarvis/b96e21edc8a1a7c076797b3e95786850). Looks like Kubernetes does not like mutiple processes which are spawned by Celery Worker. And it keeps killing the pods.   

As per the Gist, the solution was to use `--pool solo` with Celery worker. I tried it and now the Kubernetes does not kill my pods. But there is another comment which suggests to use `--pool gevent` but that would need installing `gevent` package in Docker image.    
   
So, can we install `gevent` package in Superset Docker image and use `--pool gevent`  option with celery worker by default?

**Comment by mrn3** [comment]
See also https://github.com/apache/superset/issues/26631
